src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,54 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.0
+
+### New features
+
+* Added a new `subctl uninstall` command that removes all Submariner components and dataplane artifacts, such as IP table rules, routes from
+a cluster.
+* Added a new `subctl unexport` command that stops exporting a previously exported service.
+* Added new `subctl cloud prepare` and `subctl cloud cleanup` commands for the Red Hat OpenStack Platform (RHOS).
+* New Globalnet metrics were added:
+  * Count of global Egress IPs allocated at Cluster scope, namespace scope, and for selected pods per CIDR.
+  * Count of global Ingress IPs allocated for Pods/Services per CIDR.
+* Added support for Globalnet objects verification using subctl diagnose command.
+* Added support for `--broker-namespace` flag while deploying the Broker.
+* Added support for running `subctl diagnose` on single node clusters.
+* Added support for running `subctl diagnose` from a pod in cluster.
+* `subctl cloud prepare` now deploys a dedicated gateway node as a default option on GCP and OpenStack platforms.
+* `subctl show` command now shows information about the Broker CR in the cluster.
+* `subctl gather` now collects Globalnet information.
+* `subctl diagnose` displays a warning when a generic CNI network plugin is detected.
+* New Service Discovery metric was added:
+  * Count of DNS queries handled by Lighthouse plugin.
+
+### Bug fixes
+
+* Calico is now correctly detected when used as a third-party plugin in OpenShift.
+* Services without selectors can now be resolved across the ClusterSet.
+* `subctl diagnose firewall inter-cluster` command will now work correctly for vxlan cable driver.
+
+### Internal changes
+
+* The broker token and IPsec PSK are now stored in secrets which are used in preference to the corresponding fields in the Submariner CR
+which are now deprecated. For backwards compatibility and to simplify upgrades, the deprecated fields are still populated but will be
+removed in 0.13.
+* Globalnet no longer uses `kube-proxy` chains in support of exported services. Instead, it now creates an 
+internal `ClusterIP` Service with the `ExternalIPs` set to the global IP assigned to the corresponding Service. Since some deployments of 
+Kubernetes do not allow Services with `ExternalIPs` to be created for security reasons. the pre-requisites outlined in
```

Comment:
```suggestion
Kubernetes does not allow Services with `ExternalIPs` to be created for security reasons. the pre-requisites outlined in
```

Reply:
This sentence is now quite different, but the grammar is corrected. :heavy_check_mark: